package/Dockerfile

```diff
@@ -13,6 +13,11 @@ ENV CATTLE_HELM_VERSION v2.8.0-rancher3
 ENV CATTLE_MACHINE_VERSION v0.14.0-rancher4-1
 ENV LOGLEVEL_VERSION v0.1
 
+# Add tini
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /usr/bin/tini
```

Comment:
can you follow suit and add this to the long RUN command below?  and put the ENV with the other ENV.  In general we don't like ADD of a remote resource because docker has to download it on every build to check it.

Reply:
got it